<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pay</class>
 <widget class="QDialog" name="Pay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>确认支付</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>321</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Print">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>打印信息</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>290</y>
     <width>163</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="to_order">
      <property name="text">
       <string>爷还想点餐</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="byebye">
      <property name="text">
       <string>不点了爷撤了</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
